config/locales/devise_invitable.en.yml

```diff
@@ -1,28 +1,29 @@
+---
```

Comment:
Same argument as above, the only point in having custom english files is to override the text of the english. Here we only customize `devise.edit.submit_button` over the original, so we might as well only have that key and not the whole file - see https://raw.githubusercontent.com/scambra/devise_invitable/v2.0.5/config/locales/en.yml